src/USER-REAXC/reaxc_types.h

```diff
@@ -39,6 +39,10 @@
 #include <sys/time.h>
 #include "accelerator_kokkos.h"
 
+#include "lammps.h"
```

Comment:
this is not good. you should not add includes to LAMMPS headers to reax/c headers. we only need a forward declaration here. so something like the following forward declaration should do:
```
namespace LAMMPS_NS { class Error; }
```
...and then you can use `LAMMPS_NS::Error *` and use the `error.h` include only in the .cpp file(s).